R/read.R

```diff
@@ -289,16 +298,57 @@ readH5AD <- function(file, X_name = NULL, use_hdf5 = FALSE,
     mat
 }
 
+.read_convert <- function(file, path, recursive=FALSE) {
+    .convert_special_types(rhdf5::h5read(file, path),
+                           recursive=recursive)
+}
+
+.convert_special_types <- function(obj, recursive=FALSE) {
+    if (identical(names(obj), c('categories', 'codes'))) {
+        codes <- obj[['codes']] + 1
```

Comment:
We usually use double quotes, a quick update for consistency would be great.

Reply:
Converted single to double quotes